asistente_ladm_col/logic/quality_rules/qr_boundary_points_not_covered_by_boundary_nodes.py

```diff
@@ -0,0 +1,221 @@
+                          (C) 2021 by Germán Carrillo (SwissTierras Colombia)
+        email           : contacto@ceicol.com
+                          gcarrillo@linuxmail.org
+ ***************************************************************************/
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License v3.0 as          *
+ *   published by the Free Software Foundation.                            *
+ *                                                                         *
+ ***************************************************************************/
+"""
+import processing
+from qgis.PyQt.QtCore import QCoreApplication
+
+from qgis.core import (QgsFeatureRequest,
+                       NULL)
+
+from asistente_ladm_col.config.enums import EnumQualityRuleResult
+from asistente_ladm_col.config.keys.common import (QUALITY_RULE_LADM_COL_LAYERS,
+                                                   QUALITY_RULE_ADJUSTED_LAYERS,
+                                                   ADJUSTED_INPUT_LAYER,
+                                                   ADJUSTED_REFERENCE_LAYER)
+from asistente_ladm_col.config.layer_config import LADMNames
+from asistente_ladm_col.config.quality_rule_config import (QR_IGACR1003,
+                                                           QRE_IGACR1003E01,
+                                                           QRE_IGACR1003E02,
+                                                           QRE_IGACR1003E03)
+from asistente_ladm_col.core.quality_rules.abstract_point_quality_rule import AbstractPointQualityRule
+from asistente_ladm_col.core.quality_rules.quality_rule_execution_result import QualityRuleExecutionResult
+from asistente_ladm_col.lib.geometry import GeometryUtils
+from asistente_ladm_col.logic.ladm_col.ladm_data import LADMData
+
+
+class QRBoundaryPointsNotCoveredByBoundaryNodes(AbstractPointQualityRule):
+    """
+    Check that boundary points not covered by boundary nodes
+    """
+    _ERROR_01 = QRE_IGACR1003E01
+    _ERROR_02 = QRE_IGACR1003E02
+    _ERROR_03 = QRE_IGACR1003E03
+
+    def __init__(self):
+        AbstractPointQualityRule.__init__(self)
+
+        self._id = QR_IGACR1003
+        self._name = "Los Puntos de Lindero deben estar cubiertos por nodos de Lindero"
+        self._tags = ["igac", "instituto geográfico agustín codazzi", "puntos", "punto lindero", 'lindero', "superposición"]
+        self._models = [LADMNames.SURVEY_MODEL_KEY]
+
+        self._errors = {self._ERROR_01: "Punto lindero no esta cubierto por un nodo de lindero",
+                        self._ERROR_02: "La relación topológica entre el punto lindero y el nodo de un lindero no está registra en la tabla puntoccl",
```

Comment:
"La relación topológica entre el punto lindero y el nodo de un lindero no está registrada en la tabla puntoccl"

Reply:
fixed in [1888fd7](https://github.com/SwissTierrasColombia/Asistente-LADM-COL/commit/1888fd77c8479c427aab555a9c7f73bf2027f819)